spec/errors/request_error_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'spec_helper'
-require 'yt/errors/request_error'
+require 'yt'
```

Comment:
why do we need this change?

Reply:
I think this was a change for something else I was doing. Doesn't look necessary now. Reverted in ea919e9